program/src/processor.rs

```diff
@@ -150,3 +153,18 @@ pub fn process_instruction(
         }
     }
 }
+
+/// Given a DID, validate that the signers contain at least one
+/// account that has permissions to sign transactions using the DID.
+pub fn validate_owner(did: &AccountInfo, signers: &[AccountInfo]) -> ProgramResult {
```

Comment:
nit: this could be moved to `lib.rs` for easier re-use

Reply:
fixed